src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -324,9 +324,16 @@ export default {
 				FLSS_ORIGIN_LEND_FILTER
 			);
 
+			// TODO: Remove this filter once non binary filter is ready to release
+			const nonBinaryFilterEnabled = this.$route.query?.nonBinaryFilter;
+			let genderFacets = this.allFacets?.genderFacets;
+			if (!nonBinaryFilterEnabled) {
+				genderFacets = genderFacets.filter(gender => gender.name !== 'nonbinary');
+			}
+
 			// Merge all facet options with filtered options
 			this.facets = {
-				genders: transformGenderOptions(this.allFacets?.genderFacets),
+				genders: transformGenderOptions(genderFacets),
```

Comment:
Please add a todo here with the original line comment out as well. Otherwise we'll have to look at git history to know what the revert should look like:
```
// TODO: Revert once non binary filter is ready to release
// genders: transformGenderOptions(this.allFacets?.genderFacets),
```